sftp.c

```diff
@@ -672,6 +717,21 @@ process_get(struct sftp_conn *conn, const char *src, const char *dst,
 	return(err);
 }
 
+#ifdef IRONCORE
+char *
+iron_append(const char *p1)
```

Comment:
Should you to anything to ensure that `p1` doesn't end with a `/`? Suffixing that would lead to undesirable results.